src/components/App/App.js

```diff
@@ -42,10 +45,50 @@ const App = () => {
             render={() => {
               return (
                 <>
-                  <h2>Take the Sexual Health Myth Debunking Quiz!</h2>
-                  <Link to="/quiz">YES!</Link>
+                  <div className="quiz-call-to-action">
```

Comment:
Any input on how to break this down into components is welcomed or should I have one big component called Home?